content/gamenews

```diff
@@ -1,4 +1,4 @@
-2016-10-15 20-00-00:
-	Title: A new stable release is available!
-	Author: Oliver Brakmann
-	Content: We are happy to announce a new stable release of OpenRA!\n\nA new content installer, stability improvements, especially for servers, new missions and many more interesting changes and fixes await you.\n\nHead over to our website at http://openra.net to see the full announcement, and take a look at the full changelog at http://changelog.openra.net to get an impression of all the changes.\n\nWe wish you a lot of fun with this new release!
+2016-10-19 20-00-00:
+	Title: Release 20161019 hotfix now available!
+	Author: Paul Chote
+	Content: LANDCOM 16 HQS.\nTOP SECRET\nTO: ALL FIELD COMMANDERS\n\nA SURPRISE AMBUSH OF MULTIPLAYER ISSUES\nHAS LEFT THE 20161015 RELEASE CRITICALLY\nWEAKENED.  REPORT TO www.openra.net FOR A FULL CASUALTY REPORT.\n\nPROCEEED TO www.openra.net/download/ FOR THE\nREINFORCED 20161019 RELEASE THEN CONTINUE\nTO THE MULTIPLAYER LOBBY AND DESTROY ALL\nENEMY OPPOSITION.\n\nTRANSMISSION ENDS.
```

Comment:
We could perhaps remove the "LANDCOM 16 HQS." line if you think the bottom margin is important.